msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/DiscussionForumBean.java

```diff
@@ -540,15 +540,13 @@ public String getOpenDate(){
 	}
 
 	public void setOpenDate(String openDateStr){
-		if (StringUtils.isNoneBlank(openDateStr, openDateISO)) {
+		if (StringUtils.equals(openDateStr, openDateISO)) {
```

Comment:
This check for equals seems odd. 

What about just checking isNotBlank(openDateISO)?

Reply:
I changed and also add some logic to avoid leave the dates blank when editing and ISO dates are not setted properly...